ocaml/lambda/printlambda.ml

```diff
@@ -88,6 +88,10 @@ let alloc_mode = function
   | Alloc_heap -> ""
   | Alloc_local -> "local"
 
+let alloc_mode' = function
```

Comment:
Would you mind renaming this to `alloc_mode` and then renaming the one above to something like `alloc_mode_if_local`? I really dislike `'` names.

Reply:
Fixed